src/logging.cc

```diff
@@ -402,6 +402,13 @@ static bool SendEmailInternal(const char*dest, const char *subject,
 base::Logger::~Logger() {
 }
 
+#ifdef GLOG_ENABLE_CUSTOM_PREFIX
+namespace  {
+  // Optional user-configured callback to print custom prefixes.
+  static PrefixAttacherFunction prefix_attacher_callback = NULL;
```

Comment:
```suggestion
  PrefixAttacherFunction prefix_attacher_callback = NULL;
```
`static` is not necessary in an anonymous namespace.